versions

```diff
@@ -0,0 +1,8 @@
+# An attempt to centralize version numbers consumable with bash.
+MAJOR=1
```

Comment:
Would it be better to add some s2n value to this?
To disambiguate?  Or it that not how people use these?

Reply:
Nice trying me to use the wrong project name...this was carried over from Cmake, I've made it more specific and then imported these ENV into Cmake variables.